docs/REST API/rest-api.md

```diff
@@ -1359,24 +1287,538 @@ Note that when making changes to the `integrations` object, all integrations mus
+| enable_software_vulnerabilities   | boolean | Whether or not Zendesk integration is enabled for software vulnerabilities. Only one vulnerability automation can be enabled at a given time (enable_vulnerabilities_webhook and enable_software_vulnerabilities). |
+| enable_failing_policies           | boolean | Whether or not Zendesk integration is enabled for failing policies. Only one failing policy automation can be enabled at a given time (enable_failing_policies_webhook and enable_failing_policies). |
+| url                               | string  | The URL of the Zendesk server to integrate with. |
+| email                             | string  | The Zendesk user email to use for this Zendesk integration. |
+| api_token                         | string  | The Zendesk API token to use for this Zendesk integration. |
+| group_id                          | integer | The Zendesk group id to use for this integration. Zendesk tickets will be created in this group. |
+
+<br/>
+
+##### integrations.google_calendar
+
+`integrations.google_calendar` is an array of objects with the following structure:
+
+| Name                              | Type    | Description   |
+| ---------------------             | ------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| domain                            | string  | The domain for the Google Workspace service account to be used for this calendar integration. |
+| api_key_json                      | object  | The private key JSON downloaded when generating the service account API key to be used for this calendar integration. |
+
+##### Example request body
+
+```json
+{
+  "integrations": {
+    "jira": [
+      {
+        "enable_software_vulnerabilities": false,
+        "enable_failing_poilicies": true,
+        "url": "https://jiraserver.com",
+        "username": "some_user",
+        "api_token": "<TOKEN>",
+        "project_key": "jira_project",
+      }
+    ],
+    "zendesk": [],
+    "google_calendar": [
+      {
+        "domain": "https://domain.com",
+        "api_key_json": "<API KEY JSON>"
+      }
+    ]
+  }
+}
+```
+
+#### mdm
+
+| Name                              | Type    | Description   |
+| ---------------------             | ------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| apple_bm_default_team             | string  | The default team to use with Apple Business Manager. **Requires Fleet Premium license** |
+| windows_enabled_and_configured    | boolean | Enables Windows MDM support. |
+| enable_disk_encryption            | boolean | _Available in Fleet Premium._ Hosts that belong to no team and are enrolled into Fleet's MDM will have disk encryption enabled if set to true. |
```

Comment:
The previous docs state that this is under macos_settings. CMIIW


enable_disk_encryption | boolean | body | mdm.macos_settings settings. Hosts that belong to no team and are enrolled into Fleet's MDM will have disk encryption enabled if set to true. Requires Fleet Premium license
-- | -- | -- | --

Reply:
Just looked @ the response on dogfood and it's at the top level now 👍